sbi/inference/snpe/snpe_base.py

```diff
@@ -580,11 +581,11 @@ def _loss(
         """
         if self._round == 0 or force_first_round_loss:
             # Use posterior log prob (without proposal correction) for first round.
-            log_prob = self._neural_net.log_prob(theta, x)
+            loss = self._neural_net.loss(theta, x)
         else:
-            log_prob = self._log_prob_proposal_posterior(theta, x, masks, proposal)
+            loss = self._log_prob_proposal_posterior(theta, x, masks, proposal)
```

Comment:
should be a minus here, no? Let's be extra cautious here to not revert the loss. If `self._log_prob_proposal_posterior(theta, x, masks, proposal)` returns just the log prob then it should be 
```python
loss = -self._log_prob_proposal_posterior(theta, x, masks, proposal)
```

Reply:
True, I will check.